runtime/darwinia/src/lib.rs

```diff
@@ -173,13 +173,13 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("Darwinia"),
 	impl_name: create_runtime_str!("Darwinia"),
 	authoring_version: 0,
-	spec_version: 20,
+	spec_version: 21,
 	impl_version: 0,
 	#[cfg(not(feature = "disable-runtime-api"))]
 	apis: RUNTIME_API_VERSIONS,
 	#[cfg(feature = "disable-runtime-api")]
 	apis: sp_version::create_apis_vec![[]],
-	transaction_version: 0,
+	transaction_version: 1,
```

Comment:
When should we bump transaction_version? Any signals or convention to follow?

Reply:
If the payload changed, bump it. Else, whatever.

I would like to follow polkadot here.